docs/apis-tools/camunda-api-rest/specifications/find-authorizations.api.mdx

```diff
@@ -34,8 +34,8 @@ import TabItem from "@theme/TabItem";
 
 Search for authorizations based on given criteria.
 
-Note that this endpoint is an alpha feature and not enabled on Camunda clusters out of the box.
-The Camunda 8 API (REST) Overview page provides further details.
+Note that this endpoint is an [alpha feature](/components/early-access/alpha/alpha-features.md) and not enabled on Camunda clusters out of the box.
```

Comment:
This seems correct. They point to current, but not `next`.

Reply:
They should point to whichever version of the docs the user is currently browsing. In these docs, that's `/next/`. When we cut a new version and these docs become `/` (the current version), it should remain within that version. When they become an older `/8.7/`, they will point to `/8.7/`. 

I might be misunderstanding your comment though, because I'm uncertain if you're identifying this as a problem or not.